### PR TITLE
Support for Xcode 9 auto export with manual signing

### PR DESCRIPTION
**Pull request by madhurig**
unit tests

### CODE REVIEW

File: Tasks/Xcode/Strings/resources.resjson/en-US/resources.resjson [hunk 1]
```diff
@@ -32,8 +32,8 @@
   "loc.input.help.exportTeamId": "The 10 digit Team ID from the Apple Developer Portal to use for this export.",
   "loc.input.label.exportOptionsPlist": "Export Options Plist",
   "loc.input.help.exportOptionsPlist": "Path to a plist file that configures archive exporting.",
-  "loc.input.label.xcode8AutomaticSigning": "Xcode 8 Automatic Signing",
-  "loc.input.help.xcode8AutomaticSigning": "Select this if you have an Xcode 8 project setup for Automatic Signing.",
+  "loc.input.label.xcode8AutomaticSigning": "Automatic Signing",
+  "loc.input.help.xcode8AutomaticSigning": "Select this if you have an Xcode 8 or Xcode 9 project setup for Automatic Signing.",
```
Comment (lkillgore) [inline Tasks/Xcode/Strings/resources.resjson/en-US/resources.resjson]: 'Xcode 8+'?  (ie. will we have to adjust this with every release of Xcode?)

Comment (madhurig) [inline Tasks/Xcode/Strings/resources.resjson/en-US/resources.resjson]: I considered that, but we have only Xcode 8 and 9 for now, so listed them.

Comment (lkillgore) [inline Tasks/Xcode/Strings/resources.resjson/en-US/resources.resjson]: 👌

File: Tasks/Xcode/task.json [hunk 2]
```diff
@@ -168,10 +168,10 @@
         {
             "name": "xcode8AutomaticSigning",
             "type": "boolean",
-            "label": "Xcode 8 Automatic Signing",
+            "label": "Automatic Signing",
             "defaultValue": false,
             "required": false,
-            "helpMarkDown": "Select this if you have an Xcode 8 project setup for Automatic Signing.",
+            "helpMarkDown": "Select this if you have an Xcode 8 or Xcode 9 project setup for Automatic Signing.",
```
Comment (DavidStaheli) [inline Tasks/Xcode/task.json]: P999: "set up" or "configured" instead of "setup"

Comment (madhurig) [inline Tasks/Xcode/task.json]: Thanks, changed to "configured"

File: Tasks/Xcode/xcode.ts [hunk 3]
```diff
@@ -369,6 +369,31 @@ async function run() {
...
+
+                        if (xcodeVersion >= 9 && !automaticSigningWithXcode && exportOptions === 'auto') {
+                            // Xcode 9 manual signing, set code sign style = manual
+                            tl.tool(plist).arg(['-c', 'Add signingStyle string ' + 'manual', exportOptionsPlist]).execSync();
+
+                            // add provisioning profiles to the exportOptions plist
+                            // find bundle Id from Info.plist and prov profile name from the embedded profile in each .app package
+                            tl.tool(plist).arg(['-c', 'Add provisioningProfiles dict', exportOptionsPlist]).execSync();
+
+                            for (let i = 0; i < embeddedProvProfiles.length; i++) {
```
Comment (madhurig) [inline Tasks/Xcode/xcode.ts]: Loop over all the embedded.mobileprovision and the matching Info.plist to get the right profile name and bundle identifiers when multiple apps are in the .xcarchive.

**Pull request merged**